examples/get-object-legal-hold.js

```diff
@@ -28,15 +28,24 @@ var s3Client = new Minio.Client({
 //Get Legalhold config
 s3Client.getObjectLegalHold('bucketName', 'objectName', {}, function(err, res) {
   if (err) {
-    return console.log('Unable to get legal hold config for the object', err)
+    return console.log('Unable to get legal hold config for the object', err.message) // Print only the message.
   }
   console.log(res)
 })
 
 //With versionId
 s3Client.getObjectLegalHold('bucketName', 'objectName', { versionId:'my-obj-version-uuid' }, function(err, res) {
   if (err) {
-    return console.log('Unable to get legal hold config for the object', err)
+    return console.log('Unable to get legal hold config for the object', err.message) // Print only the message.
   }
   console.log(res)
-})
\ No newline at end of file
+})
+
+//Promise based version:
+const bucketTaggingPromise = s3Client.getObjectLegalHold('bucketName', 'objectName', { versionId:'my-obj-version-uuid' })
+bucketTaggingPromise.then((data) => {
```

Comment:
copy/paste issue
`bucket`~__Tagging__~`Promise` =` bucket`__Legalhold__`Promise`

Reply:
Thank you for pointing this out. I have renamed the variable.